e2e-tests/native/capabilities.ts

```diff
@@ -10,15 +10,16 @@ const browserstackCaps = (
   const isCi = !!process.env.CI
   const prefix = isCi ? 'IG CI' : 'IG DEV'
   const app = process.env.E2E_BROWSERSTACK_APP
+
   return {
     'bstack:options': {
       buildName: `${prefix}: ${getGitBranch()}`,
       sessionName: `${platformName.toLowerCase()}: ${getGitHeadReference()}`,
       projectName: 'integreat-app-native',
       debug: true,
       realMobile: isCi,
-      networkProfile: '4g-lte-good',
```

Comment:
was an experiment to get rid of timeouts in Android, which did not work, therefore I removed it.